components/offscreen/offscreen.js

```diff
@@ -5,7 +5,7 @@ const offscreenStyleDeclarations = css`
 		direction: var(--d2l-document-direction, ${document.dir === 'rtl' ? css`rtl` : css`ltr`});
 		height: 1px;
 		inset-inline-start: -10000px;
-		left: -10000px;
```

Comment:
Wondering if we should at least leave `left: -10000px`... removing the RTL stuff will have minimal impact for old browsers, but removing this will cause the text to become visible.

Reply:
The line below is just making it `left` or `right` based on the initial `document.dir`

Reply:
Ah nice, missed that!